ts/features/itwallet/common/components/discoveryBanner/ItwDiscoveryBanner.tsx

```diff
@@ -53,7 +53,8 @@ export const ItwDiscoveryBanner = ({
   const handleOnPress = () => {
     trackItWalletBannerTap(trackBannerProperties);
     navigation.navigate(ITW_ROUTES.MAIN, {
-      screen: ITW_ROUTES.DISCOVERY.INFO
+      screen: ITW_ROUTES.DISCOVERY.INFO,
+      params: {}
```

Comment:
The upgrade banner should pass the `isL3` flag
```suggestion
      params: { isL3: true }
```